CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [2.33.1] - 2022-12-16
```

Comment:
Updating the status and the message on the CR might worth mentioning, too.

Reply:
Yea, makes sense, let me add it